### PR TITLE
Update cfmodule.json

### PR DESCRIPTION
**Pull request by KrunchMuffin**
added cfscript example and removed that script is Lucee only

### CODE REVIEW

File: data/en/cfmodule.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
 	"name":"cfmodule",
 	"type":"tag",
 	"syntax":"<cfmodule>",
+	"script":"cfmodule(template=\"nameoftemplate.cfm\",attributeCollection{});",
```
Comment (KamasamaK) [inline data/en/cfmodule.json:5]: Isn't `attributeCollection` supposed to be followed by `=`?

**Comment by KrunchMuffin** [comment]
Yup, that's what I get for copying from adobe website.

File: data/en/cfmodule.json (hunk 1 above)
Comment (KrunchMuffin) [reply data/en/cfmodule.json:5]: Yup, that's what I get for copying from adobe website.

**Comment by pfreitag** [comment]
Thanks @KrunchMuffin

**Pull request merged**